src/main/java/com/google/cloud/tools/appengine/api/deploy/DefaultStageFlexibleConfiguration.java

```diff
@@ -38,6 +39,15 @@ public void setAppEngineDirectory(File appEngineDirectory) {
   }
 
   @Override
+  public boolean getIncludeOptionalConfigurationFiles() {
+    return includeOptionalConfigurationFiles;
+  }
+
+  public void setIncludeOptionalConfigurationFiles(boolean includeOptionalConfigurationFiles) {
```

Comment:
It's a bit strange to think of this as a boolean. Also maybe not too extensible. Would it make sense to ask clients to provide the list of files they want to include?